.gitignore

```diff
@@ -12,3 +12,5 @@ coverage-final.json
 coverage-final.lcov
 npm-debug.log
 yarn.lock
+/src/styles/dgrid.css.d.ts
```

Comment:
`tcm` creates this file. Can we do a glob for `.css.d.ts` that will still include `.m.css.d.ts`?